data.json

```diff
@@ -21,7 +21,8 @@
     {"name": "Harbin", "address": "4801 Saint-Laurent"},
     {"name": "Boucherie Lawrence's", "address": "5237 Saint-Laurent"},
     {"name": "B&M", "address": "120 Saint-Viateur O"},
-    {"name": "MAÏS", "address": "5439 Saint-Laurent"}
+    {"name": "MAÏS", "address": "5439 Saint-Laurent"},
+    {"name": "Euro Déli Batory", "address": "115 Saint-Viateur O"}
```

Comment:
That's how it's written on Google Maps but it conflicts with what's written on the actual vitrine.

<img width="450" alt="screen shot 2016-04-21 at 7 40 14 pm" src="https://cloud.githubusercontent.com/assets/3929133/14727442/e4ff2694-07f8-11e6-8ec4-2436f766f66a.png">

So I'm a bit confused what to choose. Halp.